paf-mvp-client-express/src/client-node.ts

```diff
@@ -450,6 +463,37 @@ export class ClientNode extends Node {
     }
   };
 
+  verifySeed = (req: Request, res: Response, next: NextFunction) => {
+    const message = fromDataToObject<SeedSignatureContainer>(req.body);
+    try {
+      const isResponseValid = this.client.verifySeed(message);
+      if (!isResponseValid) {
+        // TODO [errors] finer error feedback
+        const error: NodeError = {
+          type: 'VERIFICATION_FAILED',
+          details: '',
+        };
+        this.logger.Error(jsonProxyEndpoints.verifySeed, error);
+        res.status(400);
```

Comment:
No need to set the status and body. It is the responsibility of the errorHandler. You can just call `next(error); `